signer/openrpc.json

```diff
@@ -145,43 +97,158 @@
     {
       "name": "jade_newAccount",
       "summary": "Creates a new account and stores it locally as a passphrase-encoded keystore file.",
-      "params": [],
+      "params": [
+        {
+          "name": "newAccount",
+          "description": "Creates a new account and stores it locally as a passphrase-encoded keystore file.",
+          "schema": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "account name",
+                "type": "string"
+              },
+              "description": {
+                "description": "account description",
+                "type": "string"
+              },
+              "passphrase": {
+                "$ref": "#/components/schemas/Passphrase"
+              }
+            }
+          }
+        },
+        {
+          "$ref": "#/components/schemas/Additional"
+        }
+      ],
       "result": {
         "name": "newAccountResult",
         "schema": {
-          "type": "boolean"
+          "$ref": "#/components/contentDescriptors/Address"
```

Comment:
this is wrong. cant `$ref` a content descriptor inside a `schema: {}`. oops.

Reply:
wtb tests :)

Reply:
https://github.com/etclabscore/jade-signer-rpc/issues/19